src/www/app/outline_server_repository/server.ts

```diff
@@ -78,6 +84,16 @@ export class OutlineServer implements Server {
 
   async connect() {
     try {
+      if (this.isDynamic) {
+        this.sessionConfig = await (
+          await fetch(this.accessKey, {
+            headers: {
+              'Content-Type': 'application/json',
```

Comment:
I'm pretty sure these aren't all the request headers we wanna be sending. What am I missing?

Reply:
Also, are we sure we want the API to just be a root object containing the sessionConfig? Or maybe the object:

```
{
  sessionConfig: {
    // session config details
  }
}
```